Cannot import css from other modules

Works in 0.8.0, broken in 1.0.2. The following error is reported by Browserify

When module is installed:

     Cannot find module './....cssify' from 
        'D:\Users\{user}\code\report-server\node_modules\@co\module'

When module is linked:

    Cannot find module './....\report-server\node_modulescssify' from
        'D:\Users\{user}\code\@co\module'

It seems to be resolving the path incorrectly...

This is being imported globally, e.g.

    var b = browserify().transform(cssify, { global: true});

Obviously I am on windows but it also doesn't work on our linux dev server so doubt it's related.

I try to require [normalize.css](https://www.npmjs.com/package/normalize.css) with cssify

```
require('normalize.css'); // normalize.css not found
// Error: Cannot find module 'normalize.css' from ...

require('normalize.css/normalize.css'); 
// html {
//     ^
// ParseError: Unexpected token

--- a/lib/process-css.js
+++ b/lib/process-css.js
@@ -32,6 +32,10 @@ function wrapCss (fileName, css, options, map) {
   var packagePath = path.join(__dirname, '..')
   var dirName = path.dirname(fileName)
   var requirePath = path.relative(dirName, packagePath)
+  
+  // On Windows, path.relative returns unescaped backslashes and 
+  // that causes cssify to not be findable.
+  requirePath = requirePath.replace(/\\/g, '/');
 
   var moduleSource = options['auto-inject']
     ? [

